app/assets/javascripts/snowcrash/modules/index_table.js.coffee

```diff
@@ -0,0 +1,157 @@
+    @loadColumnState()
+    @showHideColumns()
+
+    # -------------------------------------------------- Install event handlers
+    # We're hooking into Rails UJS data-confirm behavior to only fire the Ajax
+    # if the user confirmed the deletion
+    @$jsTable.on('confirm:complete', '#delete-selected', @onDeleteSelected)
+
+    # Handle the showing / hiding of table columns
+    @$columnMenu.find('a').on 'click', @onColumnPickerClick
+
+    # Checkbox behavior: select all, show 'btn-group', etc.
+    $('.js-index-table-select-all').click (e)=>
+      $allCheckbox = $(this).find('input[type=checkbox]')
+      isChecked = $allCheckbox.prop('checked')
+      if e.target != $allCheckbox[0]
+        isChecked = !isChecked
+        $allCheckbox.prop('checked', isChecked)
+
+      $("#{@checkboxSelector}:visible").prop('checked', isChecked)
+
+    # when selecting standalone items, check if we must also check 'select all'
+    $(@checkboxSelector).click ->
+      _select_all = $(this).prop('checked')
+
+      if _select_all
+        $(@checkboxSelector).each ->
+          _select_all = $(this).prop('checked')
+          _select_all
+
+      $('.js-index-table-select-all > input[type=checkbox]').prop('checked', _select_all)
+
+    # when selecting items or 'select all', refresh toolbar buttons
+    @$jsTable.on('click', ".js-index-table-select-all, #{@checkboxSelector}", @refreshToolbar)
+
+  loadColumnState: =>
+    if Storage?
+      @selectedColumns = JSON.parse(localStorage.getItem(@storageKey()))
+    else
+      console.log "The browser doesn't support local storage of settings."
+
+    @selectedColumns ||= ['title', 'created', 'updated']
+
+    that = this
+
+    @$columnMenu.find('a').each ->
+      $link = $(this)
+      if that.selectedColumns.indexOf($link.data('column')) > -1
+        $link.find('input').prop('checked', true)
+
+  resetToolbar: =>
+    $('.js-index-table-actions').css('display', 'none')
+    $('.js-index-select-all #select-all').prop('checked', false)
+
+  onColumnPickerClick: (event) =>
+    $target = $(event.currentTarget)
+    val     = $target.data('column')
+    $input  = $target.find('input')
+
+    if ((idx = @selectedColumns.indexOf(val)) > -1)
+      @selectedColumns.splice(idx, 1)
+      setTimeout ->
+        $input.prop('checked', false)
+      , 0
+    else
+      @selectedColumns.push(val)
+      setTimeout ->
+        $input.prop('checked', true)
+      , 0
+
+    $(event.target).blur()
+    @saveColumnState()
+    @showHideColumns()
+    false
+
+  onDeleteSelected: (element, answer) =>
+    if answer
+      that   = this
+      issueIds = []
+
+      $('.js-items-table').find(@selectedItemsSelector).each ->
+        $row = $(this).parent().parent()
+        $($row.find('td')[2]).replaceWith("<td class=\"loading\">Deleting...</td>")
+        issueIds.push($(this).val())
+
+      $.ajax @$jsTable.data('destroy-url'), {
+        method: 'POST'
```

Comment:
`DELETE` ?

Reply:
I actually wasn't sure what the right method for this is so I checked the other API's. AWS uses `POST` to delete multiple objects too: http://docs.aws.amazon.com/AmazonS3/latest/API/multiobjectdeleteapi.html. Although, I think what we choose to use won't matter too much.